.github/workflows/ci.yaml

```diff
@@ -177,10 +177,49 @@ jobs:
             --retry-max-time 60 \
             --retry-connrefused  | grep "200 OK")
           if [ ! "$exporter_ok" ]; then
-            echo "ERROR: Prometheus exporter unavailable. Aborting..." >&2
+            echo "ERROR: Prometheus exporter unavailable (connected to OSD on HTTP). Aborting..." >&2
             exit 1
           fi
 
+          echo "Successfully connected to the Exporter service"
+
+          sudo snap set opensearch-dashboards scheme=https
+          sudo snap restart opensearch-dashboards
+          sleep 10 
+
+          # Wait until the service esteblishes
+          exporter_ok=$(curl -i ${URL} \
```

Comment:
I think we should test on `https://` not `http` here, we can skip cert verification with `curl -sk`

Reply:
We can not test https for the exporeter, as it is not supporting https :-(

It is the Dashboards that is serving either HTTP or HTTPS -- which we must follow with the exporter to get a valid response.